src/components/app/details/triggerView/triggerViewConfigDiff/TriggerViewConfigDiff.tsx

```diff
@@ -400,8 +432,17 @@ export default function TriggerViewConfigDiff({
                 <div className="p-16 dc__overflow-scroll">
                     {renderConfigValuesDiff()}
                     <div className="en-2 bw-1 br-4">
-                        <div className="code-editor-header-value flex left pt-8 pb-8 pl-16 pr-16 fs-13 fw-6 lh-20 cn-9 bcn-0 dc__top-radius-4 dc__border-bottom">
-                            {editorValues.displayName}
+                        <div className="code-editor-header-value left pt-8 pb-8 pl-16 pr-16 fs-13 fw-6 lh-20 cn-9 bcn-0 dc__top-radius-4 dc__border-bottom">
+                            <span>{editorValues.displayName}</span>
+                            {isVariablesAvailable ? (
```

Comment:
Please rewrite this as `{isVariablesAvailable && (`

Reply:
resolved